pygmt/tests/test_clib_to_numpy.py

```diff
@@ -152,3 +159,82 @@ def test_to_numpy_pandas_series_numpy_dtypes_numeric(dtype, expected_dtype):
     result = _to_numpy(series)
     _check_result(result, expected_dtype)
     npt.assert_array_equal(result, series)
+
+
+########################################################################################
+# Test the _to_numpy function with PyArrow arrays.
+#
+# PyArrow provides the following dtypes:
+#
+# - Numeric dtypes:
```

Comment:
Minor detail, but PyArrow calls them types instead of dtypes, e.g. at https://arrow.apache.org/docs/python/data.html#type-metadata. So should we call them types instead of dtypes?

```suggestion
# PyArrow provides the following types:
#
# - Numeric types:
```

Reply:
Yes, I also noticed the difference. I'm OK with using `type` in the comments, but for the tests, currently, the test name is:
```
test_to_numpy_pyarrow_array_pyarrow_dtypes_numeric(dtype, expected_dtype):
```
Strictly speaking, it should be:
```
test_to_numpy_pyarrow_array_pyarrow_types_numeric(type, expected_type):
```
But I feel the minor changes in the tests are unnecessary.

Reply:
Yeah, I think they use 'type' in the comments, but use 'dtype' in the function name to be consistent with the other unit tests.

Reply:
OK, in 111738c446cdd8a1f9bc829c4896272d2ca2fc04, I've updated 'dtype' to 'type' for comments about PyArrow and left the tests unchanged.